huggle/mainwindow.cpp

```diff
@@ -3371,8 +3371,7 @@ void Huggle::MainWindow::on_actionPost_a_custom_message_triggered()
         Syslog::HuggleLogs->ErrorLog(_l("custommessage-none"));
         return;
     }
-    if (this->fCustomMessageForm != nullptr)
-        delete this->fCustomMessageForm;
+    this->setAttribute(Qt::WidgetAttribute::WA_DeleteOnClose);
```

Comment:
That's not correct.

My idea was to remove the global pointer `fCustomMessageForm` from MainWindow class and instead, add `this->setAttribute(Qt::WidgetAttribute::WA_DeleteOnClose);` to constructor of `CustomMessage` class.

That way you can instantiate multiple CustomMessage classes and you won't have to care about their deletion, because they will delete themselves once user close them (or when they close themselves, that is why you changed `hide()` to `close()`. This way the memory management would be semi-automatic and you would also give users the option to open multiple such forms in same time (for different users, for example), so they could be editing multiple messages in same time.

Reply:
Ok for the `setAttribute` in the constructor, but I don't see how you don't want to use `fCustomMessageForm` anymore.